livy/client.py

```diff
@@ -82,7 +83,9 @@ def create_session(self, kind: SessionKind) -> Session:
                 f'this version (should be one of {valid_kinds})'
             )
 
-        data = self._client.post('/sessions', data={'kind': kind.value})
+        data = self._client.post(
+            '/sessions', data={'kind': kind.value, 'conf': spark_conf or {}}
+        )
```

Comment:
I would rather not send an object here at all rather than an empty dictionary, since it corresponds more closely to the interpretation of `None`. Could we do instead:

```python
body = {'kind': kind.value}
if spark_conf is not None:
    body['conf'] = spark_conf
```